session/tlv.go

```diff
@@ -58,17 +67,6 @@ func SerializeSession(w io.Writer, session *Session) error {
 		),
 	}
 
-	if session.Macaroon != nil {
```

Comment:
Re backwards compat, would appear we always need to continue reading this. 

With the way TLV works, eventually the new clients just stop writing it and the logic checks to see which of the two have been populated.